src/netaddress.cpp

```diff
@@ -570,11 +564,6 @@ std::string CService::ToString() const
     return ToStringIPPort();
 }
 
-void CService::SetPort(unsigned short portIn)
```

Comment:
Setting ports seems to be reasonably useful part of the API, unless all of CService is made immutable I'd suggest keeping this.